bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -22,10 +22,13 @@
 #include "PreProcessor.hpp"
 #include "MsgReceiver.hpp"
 #include "RequestHandler.h"
+#include "ReservedPagesClient.hpp"
 #include <condition_variable>
 #include <memory>
 #include <mutex>
 
+bftEngine::IReservedPages *bftEngine::ReservedPagesClientBase::res_pages_ = nullptr;
```

Comment:
Make sure it doesn't create problems with the product. A few weeks ago we had issues with global static variables and eventually, we forced to define them on the stack

Reply:
it's a class static variable